tests/test_reify.cpp

```diff
@@ -0,0 +1,122 @@
+#include <catch2/catch_test_macros.hpp>
+#include <potassco/aspif_text.h>
+#include <potassco/reify.h>
+
+#include <sstream>
+
+namespace Potassco::Test::Reify {
+bool read(std::stringstream& in, std::stringstream& out, bool scc = false, bool step = false) {
+    Reifier prg(out, scc, step);
+    AspifTextInput parser(&prg);
+    return readProgram(in, parser) == 0;
+}
+
+TEST_CASE("program", "[program]") {
```

Comment:
For easier testing, it is better to split into separate test cases. A fixture might help to write this nicely.